src/contingency-flow.js

```diff
@@ -257,6 +59,6 @@ function start(url : string) : ZalgoPromise<Object> {
 }
 
 export default {
```

Comment:
Note that this `export default` approach is still needed for the unit tests with how [testdouble.js](https://github.com/testdouble/testdouble.js/) works.